src/Repository/ConferenceRepository.php

```diff
@@ -101,6 +101,21 @@ public function getNullCoordinatesConferences(): ?\Generator
         }
     }
 
+    public function findExistingConference(Conference $conference): ?Conference
+    {
+        return $this->createQueryBuilder('c')
+            ->andWhere('c.startAt = :findStartAt AND c.endAt = :findEndAt')
```

Comment:
it's a bit weird to name parameter `findXXX`, you can use `startAt`, `endAt`, and `name` instead.
BTW, you are using 2 different form with where AND. Could you use only one form?  (all condition in the same `andWhere` or 3 `andWhere`, up to you)